packages/rum-react/src/get-apm-route.js

```diff
@@ -48,7 +61,16 @@ function getApmRoute(apm) {
        */
       if (initial || pathChanged) {
         return {
-          apmComponent: withTransaction(path, 'route-change')(component),
+          apmComponent: withTransaction(
+            path,
```

Comment:
Check the path for array here as well.

Reply:
Its not useful as it will always be array if its passed as array and `match` would always be null.